packages/runtime/src/widgets/Charts/index.tsx

```diff
@@ -67,9 +67,7 @@ export const Chart: React.FC<ChartProps> = (props) => {
   }
 
   return (
-    <div ref={rootRef} style={{ height: "100%", width: "100%" }}>
```

Comment:
What issue was this causing? The intention was that this should expand to the parent container and all Chart elements would size relative to this.

Reply:
it was causing a visual issue in one of atlas dashboard design requirements. I tried to fix it for that but wasn;t working unless I removed it. Let me see if I can accommodate it as well.

Reply:
@evshi i rechecked and here is the problem. 
- We can either leave it as is. 
  - which takes all the space if its a child of any container like Row, the other child unless specified its width is squeezed. 
<img width="454" alt="Screenshot 2023-11-14 at 8 42 59 PM" src="https://github.com/EnsembleUI/ensemble-react/assets/62849614/925890cb-9393-4dd2-9fc6-c1f665854bbf">

```yaml
                    - Column:
                        styles:
                          width: 60%
                        gap: 2
                        children:
                          - Text:
                              text: Current Month Activity
                              styles:
                                fontSize: 14
                                fontWeight: 400
                                color: '#747184'
                          - Text:
                              id: totalEarning
                              styles:
                                fontSize: 24
                                fontWeight: 700
                                color: '#1D1C21'
                          - Row:
                              gap: 5
                              children:
                                - Text:
                                    id: currentPercentage
                                    styles:
                                      fontWeight: 500
                                      fontSize: 14
                                      color: '#0E9F6E'
                                - Text:
                                    text: vs Last Month
                                    styles:
                                      fontWeight: 400
                                      fontSize: 14
                                      color: '#747184'
                    - Chart:
                        type: line
                        title: ''
                        labels:
```
Now the solution in this case is to provide the chart width of 40% as other child already has 60% space assigned. Becuase the default width for chart in my case is 365px so when collective size of container is lesser it just explodes and looks horrible. 
Any suggestions for this